examples/openIMIS/createTestCustomParameters.py

```diff
@@ -0,0 +1,39 @@
+import requests
```

Comment:
@dborowiecki  can we keep name in CamelCase or do we want to use snake_case in *.py and *.json files too?

Reply:
We can keep it.